README.md

```diff
@@ -1,13 +1,45 @@
-# ShyBye
+# ShyBye #
+**Our mission** is to be a place for individuals overcome social anxiety and
```

Comment:
Missing a small word here-- to :) 

 **Our mission** is to be a place for individuals (to) overcome social anxiety and

Reply:
Actually, what do you think of this rephrase below? The word place in the original version was bugging me and sounds like a physical place since we later say in-person interaction... 

**Our mission** is to help individuals overcome social anxiety and loneliness by gamifying primarily in-person social interactions through a variety of fun social challenges

Reply:
Oh, good catch! And I like the rephrase there, I'll go ahead and make that change